pom.xml

```diff
@@ -49,4 +49,29 @@
             <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
         </repository>
     </repositories>
+
+
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>com.github.eirslett</groupId>
```

Comment:
Ultimately this should be moved to parent pom so it can be reused in eclipse/che for its npm dependencies too (dashboard and workspace-loader). But for now it's simpler to prove it works in che-lib. :D

Reply:
It requires an Eclipse CQ (test and dependency one) to introduce a new plug-in first.

Reply:
Per Roland Grunberg, keeper of all things Orbit, "I believe CQs apply only if the content is to be distributed by Eclipse foundation as well."

So you're probably fine to USE and not REDISTRIBUTE these plugins in your build.

Reply:
https://wiki.eclipse.org/Development_Resources/IP/Test_and_Build_Dependencies

it's called workswith CQs

Reply:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=12315 already approved for v1.3
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=17148 submitted for v1.6

Reply:
CQ now requires PMC approval. Note that I had to submit it for WTP, as the Portal wouldn't let me open a CQ for Che, because I'm not a committer. So, can someone from the Che PMC approve the CQ? 
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=17148